sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -316,14 +344,9 @@ export class ManagementClient extends LinkEntity {
    * @param {number} [messageCount] The number of messages to retrieve. Default value `1`.
    * @returns Promise<ReceivedSBMessage[]>
    */
-  async peek(messageCount?: number, associatedLinkName?: string): Promise<ReceivedMessageInfo[]> {
+  async peek(messageCount?: number): Promise<ReceivedMessageInfo[]> {
```

Comment:
Are these breaking changes?

Reply:
These methods are on an internal class, so this is not a breaking change